include/API/Extended.hpp

```diff
@@ -218,10 +218,16 @@ class Extended : public Core {
   void sendTransfer() const;
 
   /*
-   * Wrapper function that does attachToTangle and finally, it broadcasts and stores the
-   * transactions locally.
+   * Wrapper function that gets transactions to approve, attaches to Tangle, broadcasts and stores.
+   *
+   * @param trytes             The trytes.
+   * @param depth              The depth.
+   * @param minWeightMagnitude The minimum weight magnitude.
+   * @return Transactions objects.
    */
-  void sendTrytes() const;
+  std::vector<Transaction> sendTrytes(const std::vector<Type::Trytes>& trytes,
+                                      const unsigned int&              depth,
+                                      const unsigned int               minWeightMagnitude) const;
```

Comment:
sounds inconsistent, const uint& for depth, const uint (no ref) for minWeight.

Reply:
Good catch